gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
-awsSdk2 = { strictly = '2.28.29' }
+awsSdk2 = { strictly = '2.29.17' }
 junit = { strictly = '5.11.3' }
 assertj = { strictly = '3.26.3' }
 log4j = { strictly = '2.24.1' }
 slf4j = { require = '2.0.16' }
 lambda-log4j2 = { strictly = '1.6.0' }
-jackson = { strictly = '2.18.0' }
+jackson = { strictly = '2.18.1' }
```

Comment:
Be careful here, they have made this incompatible with some plugins (like Zalando problem — specifically custom properties)